en/release-notes/upgrade-guide-v2.0.md

```diff
@@ -36,32 +36,32 @@ List of known issues:
+- __Wrap Mode:__ Loop mode, which determines how the texture is sampled when uv exceeds 1.
+  - __Clamp:__ the value of __uv__ is automatically limited to __0, 1__ and exceeds __0 or 1__ directly.
+  - __Repeat:__ When over, the value of __uv__ is modulo, so that the texture can be rendered in a loop.
+- __Filter Mode:__ Filter mode, which determines whether to blend the surrounding pixels with the surrounding pixels when floating point samples are used to achieve the smoothing effect of texture scaling. In effect, Trilinear smoothness is higher than Bilinear, higher than Point, but Point is very suitable for pixel-style games. When scaling textures, the pixel boundaries will not be blurred, maintaining the original pixel painting style.
+  - __Point (nearest point sampling):__ directly use the nearest pixel on the uv value
+  - __Bilinear (secondary linear filtering):__ take the average of the pixel corresponding to uv and the surrounding four pixels
+  - __Trilinear (triangular linear filtering):__ Based on the quadratic linear filtering, the quadratic linear filtering results of two adjacent mipmaps are taken for the mean calculation.
+- __Premultiply Alpha:__ This is a new parameter in 2.0. When checked, the engine will enable the GL pre-multiply option during the upload of the GPU map. This is very helpful for some textures that need to be pre-multiplied. Often there are some users who can't understand the inexplicable white edges around the texture or around the text, which is caused by the semi-transparent pixels around the texture:
 
-![Spine 骨骼接缝处的奇怪白边](upgrade-guide-v2.0/spine-border.png)
+![Spine's strange white edges at the bone seams] (upgrade-guide-v2.0/spine-border.png)
 
-这在 1.x 都需要使用代码的方式才能够消除，而在 2.0 中你只需要开启贴图的预乘选项。还需要注意的是，如果你发现这样做使得贴图变暗，那么就要将对应的渲染组件的混合模式改为 ONE，ONE_MINUS_SRC_ALPHA 即可。
+This can be eliminated by using code in 1.x, and in 2.0 you only need to turn on the pre-multiply option of the texture. It's also worth noting that if you find that this makes the texture darker, you can change the blending mode of the corresponding rendering component to ONE, ONE_MINUS_SRC_ALPHA.
 
-## 2.2 RenderComponent 组件设置
+## 2.2 RenderComponent component settings
 
-在 2.0 中，我们抽象了一个新的基础组件类：RenderComponent，所有的直接渲染组件都是继承自这个组件，比如：Sprite、Label、Graphics 等。对用户来说，最直观的改变就是，继承自它的渲染组件在属性检查器中都会包含混合模式的设置选项（Src Blend Factor & Dst Blend Factor）：
+In 2.0, we abstracted a new base component class: `RenderComponent`, and all direct rendering components are inherited from this component. These include: `Sprite`, `Label`, `Graphics`, and so on. The most intuitive change for the user is that the rendering component that inherits from it will include the __Src Blend Factor__ & __Dst Blend Factor__ in the __Property inspector__:
 
-![TiledLayer 的混合模式设置](upgrade-guide-v2.0/render-component.png)
+![TiledLayer's Mixed Mode Settings] (upgrade-guide-v2.0/render-component.png)
```

Comment:
There should be no spaces between the `render-component.png` links

Reply:
This was already fixed: https://github.com/cocos-creator/creator-docs/pull/653/commits/73c768577747ee221d6de9f72551543c6da2606e